core/systems/graphite/leaderboard_content_page.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Systems.Graphite.LeaderboardContentPage do
+  use CoreWeb, :live_view_fabric
+  use Fabric.LiveView, CoreWeb.Layouts
+  use Systems.Content.Page
+
+  alias Systems.{
+    Graphite
+  }
+
+  @impl true
+  def get_authorization_context(%{"id" => id}, _session, _socket) do
+    Graphite.Public.get_leaderboard!(id)
+  end
+
+  @impl true
+  def mount(%{"id" => id}, _session, socket) do
+    leaderboard =
+      Graphite.Public.get_leaderboard!(id, Graphite.LeaderboardModel.preload_graph(:down))
+
+    # Question: who should normally pass in the "default tab", which is now hard coded to "settings_form"
```

Comment:
In general the first tab is the default tab. Unless the active tab is stored in a cookie it should be set from outside.